Matchismo/Matchismo/View/CardView.h

```diff
@@ -0,0 +1,19 @@
+//
+//  CardView.h
+//  Matchismo
+//
+//  Created by Otto Olkkonen on 24/06/2022.
+//
+
+#import <UIKIt/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CardView : UIView
+
+- (CGFloat)cornerScaleFactor;
+- (CGFloat)cornerRadius;
```

Comment:
Slightly misleading to have a property call `cornerRadius` that doesn't relate directly to the `cornerRadius` property of the view's underlying `CALayer`.  Might be better to rename it.